src/screens/WebsocketResearchScreen/SampleChatInfoScreen.tsx

```diff
@@ -201,36 +279,23 @@ const SampleChatInfoScreen = () => {
               <View style={styles.lineTop} />
               <View style={styles.lineTop} />
               <View style={styles.users}>
-                <Text style={styles.countUser}>Participants ({channelInfo?.members?.length})</Text>
+                <Text style={styles.countUser(params?.from)}>Participants {countParticipat()}</Text>
                 <FlatList
                   testID="participants"
-                  data={channelInfo?.members}
+                  data={channelInfo?.rawJson?.channel?.members}
                   keyExtractor={(item, index) => index?.toString()}
                   renderItem={({item, index}) => (
-                    <View style={{height: normalize(72)}}>
+                    <View style={styles.parentContact}>
                       <ProfileContact
                         key={index}
                         item={item}
-                        onPress={() => onContactPressed(item)}
-                        fullname={
-                          item?.user?.isMe
-                            ? `Anonymous ${anon_user_info_emoji_name} (You)`
-                            : item?.user?.username
-                        }
+                        onPress={() => onContactPressed(item, params.from)}
+                        fullname={item?.user?.name || item?.user?.username}
                         photo={item?.user?.profilePicture}
-                        showArrow={!item?.user?.isMe}
+                        showArrow={handleShowArrow(item)}
                         userId={signedProfileId}
-                        ImageComponent={
-                          item?.user?.isMe && (
-                            <View style={{marginRight: 17}}>
-                              <AnonymousIcon
-                                color={anon_user_info_color_code}
-                                emojiCode={anon_user_info_emoji_code}
-                                size={normalize(48)}
-                              />
-                            </View>
-                          )
-                        }
+                        ImageComponent={renderImageComponent(item)}
+                        from={params?.from}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `FlatList` component has been updated. Ensure that the new props and render logic are working as expected and are properly tested.



<!-- This is an auto-generated comment by OSS CodeRabbit -->